homeassistant/helpers/entity_platform.py

```diff
@@ -689,6 +689,14 @@ async def _update_entity_states(self, now: datetime) -> None:
 )
 
 
+def get_current_platform() -> EntityPlatform:
```

Comment:
Should we name it `async_` and decorate with `@callback`?

Reply:
Done.